main.js

```diff
@@ -1,5 +1,53 @@
-const line = document.getElementsByClassName('blob-code blob-code-addition js-file-line');
+function addingImportantCss(elementId) {
+  document.getElementById(`issue_${elementId}`).style.backgroundColor='rgb(61, 0, 0)';
+  let element = document.head.appendChild(document.createElement("style"));
+  element.innerHTML = `#issue_${elementId}_link::before{
+    background-color:red;
+    content: 'Important';
+    color: white;
+    width: 12px;
+    height: 12px;
+    border: rgb(45, 0, 0);
+    border-radius: 5px;
+    margin-right: 10px;
+    padding-left: 5px;
+    padding-right: 5px;
+    padding-bottom: 2px;;}`;
+};
 
-window.addEventListener('load', () => {
-    document.body.style.color = 'red';
-  });
\ No newline at end of file
+function addingRelevantCss(elementId,) {
+  document.getElementById(`issue_${elementId}`).style.backgroundColor='rgb(86, 88, 0)';
+  let element = document.head.appendChild(document.createElement("style"));
+  element.innerHTML = `#issue_${elementId}_link::before{
+    background-color: rgb(164, 167, 0);
+    content: 'Relevant';
+    color: white;
+    width: 12px;
+    height: 12px;
+    border: rgb(45, 0, 0);
+    border-radius: 5px;
+    margin-right: 10px;
+    padding-left: 5px;
+    padding-right: 5px;
+    padding-bottom: 2px;;}`;
+};
```

Comment:
Don't make two separate functions for a very similar task. Just take the "level" as parameter in the same function and modify the colors and content of the CSS accordingly.